src/component/parser/json/converter/ShapeConverter.ts

```diff
@@ -0,0 +1,24 @@
+import { JsonConvert, JsonConverter, OperationMode, ValueCheckingMode } from 'json2typescript';
+import { AbstractConverter } from './AbstractConverter';
+import Shape from '../../../../model/bpmn/shape/Shape';
+
+//////////////////////////////////////////////////////////////
+// TODO : To move in a singleton object to use here and in the BpmnJsonParser
```

Comment:
ℹ️ ok to me to keep the TODO as 1st implementation
Let's refactor that later to avoid spending too much time in infrastructure stuffs rigth now